gptme/util/ask_execute.py

```diff
@@ -22,13 +23,17 @@
 
 # Global state
 override_auto = False
+auto_skip_count = 0
 copiable = False
 editable = False
 
 # Editable text state
 _editable_text = None
 _editable_ext = None
 
+#Compiled regex pattern
+pattern = re.compile(r'(auto)(?: (\d+))?')
```

Comment:
Call this something like `re_auto`, I don't think we need to compile it, at least not at import-time.